source/release-notes.txt

```diff
@@ -10,8 +10,31 @@ Release Notes
    :depth: 1
    :class: twocols
 
-Relational Migrator 1.1.2
--------------------------
+1.1.3 Changelog
+---------------
+
+*Released July 26, 2023*
+
+New features:
+
+- Support for connecting to a SQL Server Named Instance.
+- Collection names now show for JSON schema files created from the :guilabel:`Code Generation` screen.
+
+Bug Fixes:
+
+- Fixes an issue where data verification after a sync job would sometimes 
+  stop or show as not started.
+- Fixes an issue where JSON columns could not be migrated if they contained 
+  top-level arrays.
+- Fixes an issue where excluded fields could overwrite calculated fields 
+  with the same name.
+- Fixes an issue where calculated fields based on decimal column values 
+  would return ``NaN``.
+- Fixes an issue where some fields could be cut off in the diagram view after 
+  changing the project level key handling option.
```

Comment:
(minor tweaks)
Something like: Fixes an issue where some diagram view fields are cut off after you change the project level key handling option.